src/styles/shared/components/buttons/_config.scss

```diff
@@ -7,69 +7,104 @@ $touch-target-size: var.$touch-target-size !default;
 $color-g1: var.$color-grey-1 !default;
 $color-g2: var.$color-grey-2 !default;
 $color-g3: var.$color-grey-3 !default;
+$color-g4: var.$color-grey-4 !default;
 $color-g6: var.$color-grey-6 !default;
 $color-g7: var.$color-grey-7 !default;
 $color-gsemi: var.$color-grey-semi !default;
 $color-gmid: var.$color-grey-mid !default;
 $color-brand: var.$color-brand !default;
 $color-link-hover: var.$color-link-hover !default;
 
-// The following SASS maps define color sets for each type of button
-
-// Default colors for buttons
-$colors: (
+// Colors for labeled buttons
```

Comment:
Changes here are for clarity, variants and formatting:

* Default (but unused in any applied sense) `$colors` map removed
* All button colormaps and their variants are "unrolled" here instead of maps extending maps. It's less DRY, but also less confusing and easy to parse.

Reply:
Probably is more clear for everyone